test/assetGiveaway/fixtures.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  ethers,
+  deployments,
+  getUnnamedAccounts,
+  getNamedAccounts,
+} from 'hardhat';
+import {expect} from '../chai-setup';
+import MerkleTree from '../../lib/merkleTree';
+import helpers from '../../lib/merkleTreeHelper';
+const {createDataArrayAssets} = helpers;
+
+const ipfsHashString =
+  '0x78b9f42c22c3c8b260b781578da3151e8200c741c6b7437bafaff5a9df9b403e';
+
+import {expectReceiptEventWithArgs, waitFor} from '../utils';
+
+export const setupGiveaway = async (
+  assetType: string,
+  mint?: boolean,
+  amount?: number,
+  supply?: number,
+  to?: string
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
you should be able to type it
actually if you use the createFixture pattern I mentioned below, it should auto type